frontend/components/Login.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import Link from 'next/link';
+
+export default class Login extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      email: "",
+      password: ""
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  validateForm() {
+    return this.state.email.length > 0 && this.state.password.length > 0 // ensures that the correct input is in the sectino, modify to add autho0 verification
+  }
+
+  handleChange(event) {
+    this.setState({ [event.target.name]: event.target.value });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+  }
+
+  render() {
+    return (
+      <div className="container">
+        <form className="row" onSubmit={this.handleSubmit} style={{'marginTop': '7rem'}}>
+          <div className="col s12 offset-m2 m8 input-field">
+            <input name="email" type="email" className="validate" value={this.state.email} onChange={this.handleChange} />
+            <label htmlFor="email"> Email </label>
+          </div>
+          <div className="col s12 offset-m2 m8 input-field">
+            <input name="password" type="password" className="validate" value={this.state.password} onChange={this.handleChange} />
+            <label htmlFor="password"> Password </label>
+          </div>
+          {/* Placeholder "#" used until backend is created..... */}
+          <Link href="#"> 
+            <button className=" col s2 offset-m2 btn waves-effect waves-light" disabled={!this.validateForm}>
```

Comment:
This `validateForm` doesn't work because its a function you should use a state variable which you update in the `onChange` function

Reply:
I don't think that we're going to need the password verification code that I wrote because I want to use auth0s, just left that there as a marker for where to add it in, I'll add a comment rather than leave that code.

Reply:
> This `validateForm` doesn't work because its a function you should use a state variable which you update in the `onChange` function

Oh thanks for spotting that

Reply:
Did you mean like this?

`import React, { Component } from 'react';
import Link from 'next/link';

export default class Login extends React.Component {
  constructor(props) {
    super(props);

    this.state = {
      email: "",
      password: "",
      count:0
    };

    this.handleChange = this.handleChange.bind(this);
    this.handleSubmit = this.handleSubmit.bind(this);
  }

  validateForm() {
    return (this.state.email.length > 0 && this.state.password.length > 0 && this.state.count > 0) // ensures that the correct input is in the section, modify to add autho0 verification
  }`

Reply:
I've tested navigating via the tabs and they work, I still need to fix the login link on the home page...